gem2rpm/foreman_plugin.spec.erb

```diff
@@ -27,9 +27,13 @@ precompile_flags << ' -a' if has_apidoc
 precompile_flags << ' -s' if has_assets || has_webpack
 
 config.rules[:doc] << /\/?CHANGES/
+config.rules[:doc] << /CODE_OF_CONDUCT(\.md)?/
```

Comment:
which gems bring such a file?

Reply:
```
$ rg -l CODE_OF_CONDUCT packages
packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec
packages/foreman/rubygem-googleauth/rubygem-googleauth.spec
packages/foreman/rubygem-patternfly-sass/rubygem-patternfly-sass.spec
packages/plugins/nodejs-react-json-tree/nodejs-react-json-tree.spec
packages/foreman/rubygem-google-api-client/rubygem-google-api-client.spec
packages/foreman/rubygem-secure_headers/rubygem-secure_headers.spec
packages/foreman/rubygem-fog-rackspace/rubygem-fog-rackspace.spec
packages/foreman/rubygem-fog-openstack/rubygem-fog-openstack.spec
```

Reply:
Ah, so none of the plugins maintained in the theforeman org, all good. :+1:

Reply:
Yes, probably why it makes more sense to add it to `scl.spec.erb` and gem2rpm itself.